content/articles/nesting-navigators-and-routing-with-react-navigation-in-react-native/index.md

```diff
@@ -0,0 +1,266 @@
+
+2. An advantage for this project is familiarity with the iOS or Android simulators and setting up a new React Native environment in your development environment.
+
+There will be several steps that we will walk through to achieve our goal in this tutorial.
+
+### First Step- Application development and module installation
+
+1. Install expo-cli by running the following command in your terminal.
+
+```bash
+$ npm install -g expo-cli
+```
+
+2. The next step is to use the commands below to create a new project.
+
+```bash
+$ npx react-native init react-navigation-routing --version 0.63.2
+```
+
+3. After that, go to the newly created directory by using the following command here:
+
+```bash
+$ cd react-navigation-routing
+```
+
+4. Use the following command to install the necessary packages:
+
+```bash
+npm install –save react-navigation react-navigation-stack react-native-reanimated react-native-gesture-handler react-native-screens react-native-vector-icons
+```
+
+5. Open the iOS or Android application, depending on your device respectively using the following code:
+
+```bash
+$ npm run ios
+
+$ npm run android
+```
+
+Now we need to open the `app.js` and then replace its content. The content that we'll replace with includes two screens that we'll create later i.e the home and NewWorker screens. To open up `App.js` run the following code in the terminal:
+
+```bash
+nano App.js
+```
+
+The following code must be copied into the app.js file that is now open to be used, as previously mentioned:
+
+```js
+import 'react-native-gesture-handler';
+import React from "react";
+import { StyleSheet, Text, View } from 'react-native';
+
+import HomeScreen from "./HomeScreen";
+import NewWorkerScreen from "./NewWorkerScreen";
+import { createAppContainer } from "react-navigation";
+import { createStackNavigator } from "react-navigation-stack";
+
+
+const Stack = createStackNavigator();
+
+class App extends React.Component {
+
+  render() {
+    return (
+      <NavigationContainer>
+        <Stack.Navigator>
+          <Stack.Screen
+            name="Home"
+            component={HomeScreen}
+          />
+          <Stack.Screen
+            name="NewWorker"
+            component={NewWorkerScreen}
+          />
+        </Stack.Navigator>
+      </NavigationContainer>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    alignItems: 'center',
+    justifyContent: 'center',
+    backgroundColor: '#fff',
+  },
+});
+
+export default App;
+```
+
+### Second Step- Creating a HomeScreen and NewWorkerScreen
+
+It will be necessary for your app starting with the HomeScreen. To see how many workers are already connected to your network, go to the HomeScreen.
+
+Then, in the HomeScreen.js you will have to create a new file, paste the App.js code and open:

```

Comment:
Instead of jumping right in, Please add a section to explain what the two screens would do and give them an overview of what you are going to build.